package.json

```diff
@@ -13,6 +13,7 @@
   },
   "devDependencies": {
     "turbo": "^1.10.2",
+    "@craco/craco": "^7.1.0",
```

Comment:
This seems incorrect to me. We only use craco for Studio. Shouldn't it be in the `apps/studio/package.json` file instead? 🤔